Add billing_mode option to the DynamoDB backend so `pay_per_request` or `provisioned` billing can be configured (#27674)
gravitational/teleport: docs/pages/reference/backends.mdx

}
```

<Admonition
  type="warning"
  title="on_demand with autoscaling"
>

>   If `on_demand` is set to true, auto scaling will be disabled and read/write capacities will be set to zero

Reviewer: ```suggestion
  If `on_demand` is set to true, the settings for read/write capacity and auto scaling will be ignored.
```
I don't think we actually mention the `read_capacity_units` and `write_capacity_units` tho, should this only mention autoscaling?
